### ISSUE TITLE
Fix name error in face app

### ISSUE THREAD

**Issue by eighteyes** [matrix-io/matrix-os#31]
when running face example in `matrix-src-apps` I get a crash.
```
 device.service start> +5ms { type: 'service-init', name: 'face', options: {} }
Uncaught exception:  TypeError: Cannot read property 'name' of undefined
    at Object.registerComponent (/home/pi/matrix-os/lib/service/zeromq.js:32:45)
    at EventEmitter.start (/home/pi/matrix-os/lib/device/service.js:91:37)
    at emitOne (events.js:96:13)
    at EventEmitter.emit (events.js:188:7)
    at ChildProcess.messageHandler (/home/pi/matrix-os/lib/event/app.js:108:23)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at process.nextTick (internal/child_process.js:744:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9) TypeError: Cannot read property 'name' of undefined
    at Object.registerComponent (/home/pi/matrix-os/lib/service/zeromq.js:32:45)
    at EventEmitter.start (/home/pi/matrix-os/lib/device/service.js:91:37)
    at emitOne (events.js:96:13)
    at EventEmitter.emit (events.js:188:7)
    at ChildProcess.messageHandler (/home/pi/matrix-os/lib/event/app.js:108:23)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at process.nextTick (internal/child_process.js:744:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
UNKNOWN ERROR! TypeError: Cannot read property 'name' of undefined
    at Object.registerComponent (/home/pi/matrix-os/lib/service/zeromq.js:32:45)
    at EventEmitter.start (/home/pi/matrix-os/lib/device/service.js:91:37)
    at emitOne (events.js:96:13)
    at EventEmitter.emit (events.js:188:7)
    at ChildProcess.messageHandler (/home/pi/matrix-os/lib/event/app.js:108:23)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at process.nextTick (internal/child_process.js:744:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
  matrix DESTROYING +13ms
```

**Comment by eighteyes** [comment]
@brianofrokk3r uploaded the latest to sc/gesture, there is a merge conflict, i'll fix that tomorrow, but it has those files that were missing

**Comment by diegoribero** [comment]
Both face and gesture worked for me after the update, just had to install them from `dev`, switched back to `sc/gesture` and viola! We can test once more after conflict resolution and I think we are good to merge

**Comment by brianofrokk3r** [comment]
@diegoribero @eighteyes we should close this.

**Comment by brianofrokk3r** [comment]
Resolved in [v0.6.0](https://github.com/matrix-io/matrix-os/releases/tag/v0.6.0)
